htdocs/banners.php

```
@@ -275,55 +298,55 @@ function emailStats($cid, $bid)
+                $result = $xoopsDB->query($sql);
+                if ($xoopsDB->isResultSet($result) && $result instanceof \mysqli_result) {
+                    $row = $xoopsDB->fetchRow($result);
+                    if (false === $row) {
+                        redirect_header('banners.php', 2);
+                    }
+                    [$bid, $imptotal, $impmade, $clicks, $imageurl, $clickurl] = $row;
+                    if ($impmade == 0) {
+                        $percent = 0;
+                    } else {
+                        $percent = substr(100 * $clicks / $impmade, 0, 5);
                     }
```

PHPMD (2.15.0)

[error] The method emailStats uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)